config/params.ini

```diff
@@ -19,12 +19,13 @@ saved_games_folder=
 
 [routes]
 run_trav=0
-run_pindle=1
-run_eldritch=1
+run_pindle=0
+run_eldritch=0
 run_shenk=0
 run_nihlatak=0
+run_arcane=1
```

Comment:
Leave the default. If a user wants to run any other they will adapt it on their own or use their custom.ini